### ISSUE TITLE
atom cannot find go-debug/node_modules/fsevents/lib/binding

### ISSUE THREAD

**Issue by ghost** [lloiser/go-debug#110]
I have just installed go and delve on macOS 10.12.6, and I want to debug using atom 1.18.1, so I installed go-debug. I found issue #101 which ended with the reporter saying that removing the folder and reinstalling helped. Unfortunately this has not been the case for me.

I still get:

![go-debug](https://user-images.githubusercontent.com/29540585/29034927-cd6ad844-7b91-11e7-9da5-7de46e8d0e1e.jpg)

I have done all of the things you asked the previous reporter to do.

### Checked versions
```
$ apm --version
apm  1.18.1
npm  3.10.10
node 6.9.5 x64
python 2.7.10
git 2.11.0
```
### Installation details
I don't have node installed separately from atom, and I just installed the latest version of atom today.

I installed `go-debug` from apm.

### Done all you suggested
* remove the go-debug folder and use `apm install go-debug`.
* remove the go-debug folder and install go-debug via the atom Install settings tab.
* run `apm rebuild` inside .atom/packages/go-debug
* rebuilt inside of atom

All with the same results, I'm afraid.

Hope you can help!

**Comment by lloiser** [comment]
one more thing that you can try:
* `cd  .atom/packages/go-debug`
* `rm -r node_modules`
* `apm install`

**Comment by lloiser** [comment]
This should be fixed with the latest version 1.6.0
